Cargo.toml

```diff
@@ -4,5 +4,23 @@ resolver = "2"
 
 members = [
     "event-driven",
-    "event-driven-macros"
+    "event-driven-macros",
+    "kv_store",
+    "machine",
+    "streambed-machine",
 ]
+
+[workspace.package]
+description = "Event Driven Finite State Machine library"
+readme = "README.md"
+version = "0.7.0"
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/titanclass/edfsm.git"
+
+[workspace.dependencies]
+edfsm = { path = "event-driven", version = "0.7.0" }
+event-driven-macros = { path = "event-driven-macros", version = "0.7.0" }
+streambed-machine = { path = "streambed-machine", version = "0.1.0" }
+machine = { path = "machine", version = "0.7.0" }
+kv_store = { path = "kv_store", version = "0.7.0" }
```

Comment:
We'll also need any path based dependencies to go in here along with the version that they are to be published at. For example, in streambed:

```
streambed = { path = "streambed", version = "0.11.1" }
streambed-confidant = { path = "streambed-confidant", version = "0.11.1" }
streambed-logged = { path = "streambed-logged", version = "0.11.1" }
streambed-test = { path = "streambed-test", version = "0.11.1" }
```